pkg/cli-runtime/wait/wait_test.go

```diff
@@ -158,7 +158,7 @@ func TestUntilDelete(t *testing.T) {
 	defer func() {
 		BackOffTime = previousBackOffTime
 	}()
-	BackOffTime = 10 * time.Millisecond
+	BackOffTime = 30 * time.Millisecond
```

Comment:
why not pass `BackOffTime` as func function parameter?

Reply:
Do you mean adding Backofftime as a function parameter to UntilDelete ? 
Having the BackOfftime public enables calling function to reset the value from default.

Reply:
overriding public variable is not great way to set a variable. In future to add tests can you see how this will become a problem?

Reply:
workload delete uses the untilDelete function in wait file , and if we remove this public variable and have it accept as a function parameter, we will need to have prepare for each test case that store the backOfftime in context for each of test case and in workload delete we pull the value back from context or default to 5 second.